### ISSUE TITLE
Feature request: Close top drawer when swiping right to left

### ISSUE THREAD

**Issue by rafree** [ubports/ubuntu-touch#1903]
### Description of the feature
When I have the top drawer open (swipe from top to bottom), the gesture "swipe from right to left" seems to be deactivated. This limits the UX for me, as I sometimes try to do that and I then first need to close the drawer (swipe bottom to top) and then swipe right to left again. I would prefer if it behaves similar as the "swipe from left to right": If I do that, the top drawer closes, and the app overview appears. So if I have the top drawer open and I do a swipe from right to left, I would like to see the top drawer close and get to the open app overview.

**Comment by 7sillyeaters** [comment]
I agree that this is sort of a clumsy way to shut the top drawer, when you get used to swiping from the right hand side. Not super intuitive. Thank you.
